src/app/core/components/language-picker/language-picker.component.html

```diff
@@ -1,8 +1,17 @@
-<p-dropdown
-  *ngIf="languages && current"
-  [options]="languages"
-  [ngModel]="current"
-  optionLabel="tag"
-  [styleClass]="styleClass || 'alg-language-picker-dropdown'"
-  (onChange)="languageChanged($event)"
-></p-dropdown>
+<ng-container *ngIf="languages.length > 1; then picker else readonlyContainer"></ng-container>
+<ng-template #picker>
+  <p-dropdown
+    *ngIf="current"
+    [options]="languages"
+    [ngModel]="current"
+    optionLabel="tag"
+    [styleClass]="styleClass || 'alg-language-picker-dropdown'"
+    (onChange)="languageChanged($event)"
+  ></p-dropdown>
+</ng-template>
+
+<ng-template #readonlyContainer>
+  <span *ngIf="current">
+    {{ current.tag | uppercase }}
+  </span>
```

Comment:
For a single-language platform, I don't think that's useful to waste top-bar space for indicating the language.

Reply:
Fixed